src/elexmodel/models/BaseElectionModel.py

```diff
@@ -264,3 +267,46 @@ def get_coefficients(self):
         These coefficients are for the point prediciton only, not for the lower or upper intervals models.
         """
         return self.features_to_coefficients
+
+    def compute_lambda(
+        self,
+        reporting_units,
+        possible_lambda_values=[],
+        estimand="",
+        K=3,
+    ):
+        if (
+            len(self.features) == 0
+            or len(possible_lambda_values) == 0
+            or estimand == ""
+            or not (f"{estimand}" in reporting_units)
```

Comment:
also curious about these two checks

Reply:
1. Removed `or estimand == ""`.
2. The `or not (f"{estimand}" in reporting_units)` check is neccessary in the case that the estimand is not in the reporting_units. This seems to only happen when the estimand is "turnout." In that case, should I change this check to check if the estimand is "turnout" and go from there?